kernel/ksu.c

```diff
@@ -57,7 +57,11 @@ void escape_to_root() {
 	memset(&cred->cap_ambient, 0xff, sizeof(cred->cap_ambient));
 
 	// disable seccomp
+#ifdef CONFIG_GENERIC_ENTRY
+	current_thread_info()->flags &= ~SYSCALL_WORK_SECCOMP;
```

Comment:
should be `current_thread_info()->syscall_work`: https://elixir.bootlin.com/linux/v5.11.22/source/include/linux/thread_info.h#L135

Reply:
I thought we can also use `SYSCALL_WORK_SECCOMP` ? I can be able to compile
https://elixir.bootlin.com/linux/v5.11.22/source/include/linux/thread_info.h#L50

Reply:
I mean `current_thread_info()->syscall_work` instead of  `current_thread_info()->flags`

Reply:
> should be `current_thread_info()->syscall_work`: https://elixir.bootlin.com/linux/v5.11.22/source/include/linux/thread_info.h#L135

I'll make another PR for this